src/odemis/acq/test/fastem_test.py

```diff
@@ -116,23 +130,14 @@ def test_overview_acquisition(self):
         self.assertGreaterEqual(bbox[3], scintillator5_area[3])  # Top
 
 
+@unittest.skipIf(TEST_NOHW, "No simulator running or HW present. Skip fastem ROA tests.")
 class TestFastEMROA(unittest.TestCase):
     """Test region of acquisition (ROA) class methods."""
 
-    # Don't need the GUI, but a working backend
-    backend_was_running = False
-
     @classmethod
     def setUpClass(cls):
-        # try:
-        #     test.start_backend(FASTEM_CONFIG)  # TODO this backend will not work yet as not ASM component
-        # except LookupError:
-        #     logging.info("A running backend is already found, skipping tests")
-        #     cls.backend_was_running = True
-        #     return
-        # except IOError as exp:
-        #     logging.error(str(exp))
-        #     raise
+        if TEST_NOHW is True:
```

Comment:
Is this still needed here if you also have the `skipIf` on line 133?

Reply:
Indeed that doesn't make sense. In Odemis, we usually do it here in the setupclass, so I will remove the decorater line.